docs/guide-pseudonymisation.md

```diff
@@ -63,9 +63,19 @@ with a sequence there are two endpoints available:
 #### `GET <host>/plugin/metadata-manager/sequences`
 Returns the sequences present in the database.
 
+> Note: Keep in mind that the sequence identifiers contain number signs (#). These need to be URL 
```

Comment:
why do we do this?

Reply:
PostgreSQL has a restriction on the length of table (and sequence) names, which is a maximum of 32 characters. We don't have this restriction on entity type identifiers and attribute names. So to make sure that every identifier fits in the database, we truncate the identifier and append a hash of the identifier to make sure that it's unique. In the case of the sequence identifiers, a hash of the entity type id + attribute name is used. The format we use is:

`<truncated_id>#<hash>`

Reply:
ah ok, maybe that is good to add then. max size 32 characters, else you need to url encode